### PR TITLE
[ADMIN] Publication March 2024 Final Approval

### PR DESCRIPTION
**Pull request by gocadimic**
> Note:

**In making a submission to the Open Mobile Alliance Registry, you understand and agree that your submission is made under the BSD-3 Clause License as set forth on the Open Mobile Alliance Registry and that additional formats of your submission may be created by the Open Mobile Alliance tools and processes, which may convert the content of your submission, and that the Open Mobile Alliance bears no liability for the additional formats of your submission nor for any conversion of the content of your submission.**

### CODE REVIEW

**Review by jpradocueva — APPROVED** [review]
Ok

File: DDF.xml [hunk 1]
```diff
@@ -320,10 +319,9 @@ A LwM2M Server MUST support block-wise transfer. Other protocols, such as HTTP/H
         <TS>http://www.openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A.pdf</TS>
         <TSLink>1</TSLink>
     </Item>
-
     <Item>
         <ObjectID>8</ObjectID>
-        <URN>urn:oma:lwm2m:oma:8</URN>
+        <URN>urn:oma:lwm2m:oma:8:1.0</URN>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <URN>urn:oma:lwm2m:oma:8</URN>
```

Comment (jpradocueva) [reply DDF.xml]: The first version, v1.0, doesn't need to be added to the URN.

File: DDF.xml [hunk 2]
```diff
@@ -332,12 +330,12 @@ A LwM2M Server MUST support block-wise transfer. Other protocols, such as HTTP/H
...
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
-        <TS>http://www.openmobilealliance.org/release/LWM2M_LOCKWIPE/V1_0_3-20210119-A/OMA-TS-LWM2M_LockWipe-V1_0_2-20210119-A.pdf</TS>
+        <TS>https://www.openmobilealliance.org/release/LWM2M_LOCKWIPE/V1_0_4-20240312-A/OMA-TS-LWM2M_LockWipe-V1_0_4-20240312-A.pdf</TS>
         <TSLink>1</TSLink>
     </Item>
     <Item>
         <ObjectID>9</ObjectID>
-        <URN>urn:oma:lwm2m:oma:9</URN>
+        <URN>urn:oma:lwm2m:oma:9:1.0</URN>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <URN>urn:oma:lwm2m:oma:9</URN>
```

File: DDF.xml [hunk 3]
```diff
@@ -463,10 +459,9 @@ A LwM2M Server MUST support block-wise transfer. Other protocols, such as HTTP/H
         <TS></TS>
         <TSLink>0</TSLink>
     </Item>
-
     <Item>
         <ObjectID>14</ObjectID>
-        <URN>urn:oma:lwm2m:oma:14</URN>
+        <URN>urn:oma:lwm2m:oma:14:1.0</URN>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <URN>urn:oma:lwm2m:oma:14</URN>
```

File: DDF.xml [hunk 4]
```diff
@@ -475,13 +470,12 @@ A LwM2M Server MUST support block-wise transfer. Other protocols, such as HTTP/H
...
         <DDFLink>1</DDFLink>
-	<TS>http://www.openmobilealliance.org/release/LWM2M_SWMGMT/V1_0_2-20210119-A/OMA-TS-LWM2M_SwMgmt-V1_0_2-20210119-A.pdf</TS>
+	<TS>https://www.openmobilealliance.org/release/LWM2M_SWMGMT/V1_0_3-20240312-A/OMA-TS-LWM2M_SwMgmt-V1_0_3-20240312-A.pdf</TS>
         <TSLink>1</TSLink>
     </Item>
-
     <Item>
         <ObjectID>15</ObjectID>
-        <URN>urn:oma:lwm2m:oma:15</URN>
+        <URN>urn:oma:lwm2m:oma:15:1.0</URN>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <URN>urn:oma:lwm2m:oma:15</URN>
```

File: DDF.xml [hunk 5]
```diff
@@ -553,20 +545,21 @@ A LwM2M Server MUST support block-wise transfer. Other protocols, such as HTTP/H
     </Item>
     <Item>
         <ObjectID>19</ObjectID>
-        <URN>urn:oma:lwm2m:oma:19</URN>
+        <URN>urn:oma:lwm2m:oma:19:1.0</URN>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <URN>urn:oma:lwm2m:oma:19</URN>
```

File: DDF.xml [hunk 6]
```diff
@@ -717,7 +710,7 @@ Instances of this Object are linked from Instances of Object 0 using the OSCORE
     </Item>
     <Item>
         <ObjectID>25</ObjectID>
-        <URN>urn:oma:lwm2m:oma:25</URN>
+        <URN>urn:oma:lwm2m:oma:25:1.0</URN>
```
Comment (jpradocueva) [inline DDF.xml]:
```suggestion
        <URN>urn:oma:lwm2m:oma:25</URN>
```

**Review by jpradocueva — APPROVED** [review]
FIrst object version `v1.0`, doesn't need to be added to the URN naming.

File: DDF.xml [hunk 7]
```diff
@@ -505,10 +498,9 @@ A LwM2M Server MUST support block-wise transfer. Other protocols, such as HTTP/H
         <DDF>15.xml</DDF>
         <Vorto></Vorto>
         <DDFLink>1</DDFLink>
-        <TS>http://www.openmobilealliance.org/release/LWM2M_DevCapMgmt/V1_0_4-20201208-A/OMA-TS-LWM2M_DevCapMgmt-V1_0_4-20201208-A.pdf</TS>
+        <TS>https://www.openmobilealliance.org/release/LWM2M_DevCapMgmt/V2_0_1-20240312-A/OMA-SUP-XML_15-V2_0_1-20240212-A.xml</TS>
```
Comment (jpradocueva) [inline DDF.xml]: @gocadimic, this link must point to the TS, not the Object. You need to update the link to point to https://www.openmobilealliance.org/release/LWM2M_DevCapMgmt/V2_0_1-20240312-A/OMA-TS-LWM2M_DevCapMgmt-V2_0_1-20240312-A.pdf

**Review by jpradocueva — CHANGES_REQUESTED** [review]
One comment left behind about the pointer to the TS for Dev Capabilities.

File: DDF.xml (hunk 7 above)
Comment (gocadimic) [reply DDF.xml]: @jpradocueva, The link has been updated to point to TS.

**Pull request merged**